### ISSUE TITLE
🐛[BUG] Kirin issue

### ISSUE THREAD

**Issue by ccl085** [HorizonFFXI/HorizonXI-Issues#344]
Kirin got the unclaimed pop issue (I assume every ??? mob has it at this point ?).

Kirin can auto attack on test server, this is most likely due to a mistake on ffxiclopedia saying Kirin has earth additional effect on melee hit -> I did capture Kirin on retail before, he doesn't auto attack and just spam TP move.
-> video of kirin not meleeing I could find on youtube -> https://www.youtube.com/watch?v=THjv0B_3Dps

Tail smash/whip seems to miss the bind additional effect

Deadly Hold **MIGHT** be too damaging (was hiting me for 1.4-1.9k), 2 era source mention 500 -> https://forums.jeuxonline.info/sujet/810019/kirin-strategie
"( deadly hold 500+)"

and 704 -> https://blog.wyred.nu/2005/10/ 
"Wyred uses Ground Strike.
Kirin takes 1482 points of damage.
Skillchain: Light.
Kirin takes 185 points of damage.
Kirin uses Deadly Hold.
Wyred takes 704 points of damage.
Wyred (°∀°)"

Kirin did pop with Genbu already popped (need to check other mob that can summon add such as DL)

**Comment by xkoredev** [comment]
My thoughts on this:
- I can take the "??? spawns yellow" issue if noone else is looking into it.
- 1.4-1.9k is not so off if you were naked. But this is from fun retail memories of ls mates getting 1 shot for 1.5k or so. I have no era sources. Will look into it.
- Ground Strike for 1.5K to kirin seems a lot. What gear were you wearing?

**Comment by xkoredev** [comment]
Looks like somebody else is looking into the ??? issue, so just need to confirm the damage numbers.

**Comment by CriticalXI** [comment]
I wrote Kirin to be able to auto attack on purpose. I found him auto attacking in videos in era which I'll have to look around for again.

**Comment by ccl085** [comment]
"Ground Strike for 1.5K to kirin seems a lot. What gear were you wearing?" This is most likely a 3000 tp SA ground strike as this is a 2005 report from a japanese blog.

"I wrote Kirin to be able to auto attack on purpose. I found him auto attacking in videos in era which I'll have to look around for again.": Could it be that Kirin changed at some point? I was kinda mind blown when I saw it "couldn't" melee hit since both high level version (Qilin 99 ) and Escha Kirin (119) both can melee.

Also found this -> https://www.youtube.com/watch?v=BAJQr56v5-Y&t=143s he does open with a melee attack but I don't see a 2nd one (pixel video there might be one I can't notice)
